src/api/include/migraphx/migraphx.h

```diff
@@ -459,6 +469,13 @@ migraphx_status migraphx_quantize_int8(migraphx_program_t prog,
                                        migraphx_target_t target,
                                        migraphx_quantize_int8_options_t options);
 
+migraphx_status migraphx_context_destroy(migraphx_context_t context);
+
+migraphx_status migraphx_context_assign_to(migraphx_context_t output,
+                                           const_migraphx_context_t input);
+
+migraphx_status migraphx_context_create(migraphx_context_t* context);
```

Comment:
Ideally, we shouldn't have a create/destroy for context as users shouldn't be creating and destroying the context. Whats the reason for adding this? Is this to support `auto_convert_param`? Maybe it would be better to inherit from `handle_lookup` or add an overload for `migraphx_adl_handle_lookup`.

Reply:
Yes, it is for `auto_convert_param`.

Reply:
Made changes for `context` to directly inherit from `handle_lookup` and removed constructors, deleters for context.